arlo-client/src/types.ts

```diff
@@ -63,6 +63,7 @@ export interface IAuditBoard {
   id: string
   name: string
   members: IAuditBoardMember[]
+  offline?: boolean // TODO make non-optional
```

Comment:
@benadida or @eventualbuddha could one of you have the backend accept this from the `/jurisdictions` POST endpoint and return it with the `/status` GET endpoint?

Reply:
Added #246

Reply:
It seems to me this is the wrong place to add this field, since it's not done on a per audit board basis? Shouldn't we have a single flag on the whole audit? (Also, worried that `offline` is probably the wrong term here.)

Reply:
I think @benadida is right. Seems like it's per contest, or possibly per election. As to terminology, I'd defer to @mcchilders and @umbernhard.

Reply:
@benadida Yeah, it could be higher up in the data tree, though if it's higher than jurisdiction level we'll need a new endpoint to handle it, or have the toggle happen in the first form so it can be included in the post to `/basic`.